backend/src/main/java/com/bakdata/conquery/models/datasets/concepts/filters/specific/PostalCodeFilter.java

```diff
@@ -0,0 +1,39 @@
+package com.bakdata.conquery.models.datasets.concepts.filters.specific;
+
+import java.util.EnumSet;
+import java.util.List;
+
+import com.bakdata.conquery.apiv1.frontend.FEFilter;
+import com.bakdata.conquery.apiv1.frontend.FEFilterType;
+import com.bakdata.conquery.io.cps.CPSType;
+import com.bakdata.conquery.models.datasets.concepts.filters.Filter;
+import com.bakdata.conquery.models.datasets.concepts.filters.SingleColumnFilter;
+import com.bakdata.conquery.models.datasets.concepts.filters.postalcode.PostalCodeFilterNode;
+import com.bakdata.conquery.models.datasets.concepts.filters.postalcode.PostalCodeSearchEntity;
+import com.bakdata.conquery.models.datasets.concepts.filters.postalcode.PostalCodesManager;
+import com.bakdata.conquery.models.events.MajorTypeId;
+import com.bakdata.conquery.models.query.queryplan.filter.FilterNode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@CPSType(id = "POSTAL_CODE", base = Filter.class)
+public class PostalCodeFilter extends SingleColumnFilter<PostalCodeSearchEntity> {
+
+	@Override
+	public void configureFrontend(FEFilter f) {
+		f.setType(FEFilterType.STRING);
```

Comment:
Hier müssten wir mit @Kadrian zusammen ein Konzept überlegen, wie wir mehrere FilterType kodieren, oder ob wir das zu einer Array migrieren?

@thoniTUB hast du hierzu ideen?

Reply:
Also weil String bedeutet, das Kai das als Textfeld rendert und du dann auch in createFilterNode einen Text raus kriegst.

Reply:
ich wollte eigentlich den Typen POSTAL_CODE angeben und nicht string

Reply:
Dann bitte noch ein TODO ran für den CompoundType oder was letztendlich mit Kai abgesprochen wird

Reply:
> POSTAL_CODE 

Da wir den PLZ Filter in Eva halten wollen müsstest du das über ein Mixin oder ähnliches in Eva erreichen.

Reply:
Ich denke dass wenn der Compund-Typ definiert ist, kann dieser nach Conquery

Reply:
Ja, genau darum ging es mir :)